app/src/main/java/com/arctouch/codechallenge/core/extensions/ViewExtensions.kt

```diff
@@ -0,0 +1,11 @@
+package com.arctouch.codechallenge.core.extensions
+
+import android.view.View
+
+fun View.setVisible() {
```

Comment:
That depends on the case, normally i delete unused methods/properties but sometimes they are helpful, in this particular case i didn't remove, because i intend to keep developing this app and it's almost certain that i'm gonna need to change a view visibility, also from time to time i like to run some code analysis to find problems and dont forget to remove/fix things